### PR TITLE
chore: added custom maps by worldchanges

### PR DESCRIPTION
**Pull request by dudantas**
trainers, event room and npc shop added by worldchanges

### CODE REVIEW

File: data/movements/movements.xml [hunk 1]
```diff
@@ -56,6 +56,10 @@
 	<movevent event="DeEquip" itemid="28658" slot="necklace" script="quests/cults of tibia/begintask.lua" />
 	<movevent event="StepIn" itemid="28869" script="quests/cults of tibia/geyser.lua" />
 
+	<!-- Customs -->
+	<movevent event="StepIn" actionid="14876" script="custom/trainerTP.lua"/>
+    <movevent event="StepIn" actionid="14877" script="custom/trainerTPRemove.lua"/>
```
Comment (gpedro) [inline data/movements/movements.xml]: missing identation

File: data/globalevents/globalevents.xml [hunk 2]
```diff
@@ -10,6 +9,9 @@
     <globalevent name="Preceptor Lazare" interval="900000" script="bosses/Preceptor Lazare.lua" />
 	<globalevent name="Grand Commander" interval="900000" script="bosses/Grand Commander Soeren.lua" />
     <globalevent name="Thawing Dragon Lord" interval="900000" script="bosses/Thawing Dragon Lord.lua" />
+	
+	<!-- Custom -->
+	<globalevent name="Additional" type="startup" script="custom/worldchanges/additionals/trainers_eventRoom.lua"/>
```
Comment (gpedro) [inline data/globalevents/globalevents.xml]: you can rename for custom/additionalsMapLoading.lua, what's your think?

File: data/movements/scripts/custom/trainerTPRemove.lua [hunk 3]
```diff
@@ -12,7 +12,7 @@ local function removeTrainers(position)
 		end
 	end
 end
-
+
```
Comment (gpedro) [inline data/movements/scripts/custom/trainerTPRemove.lua]: trailing spaces

File: data/movements/scripts/custom/trainertp.lua [hunk 4]
```diff
@@ -52,7 +52,7 @@ local function calculatingRoom(uid, position, coluna, linha)
 		if isBusyable(room_pos) then
 			player:teleportTo(room_pos)
 			player:getPosition():sendMagicEffect(CONST_ME_TELEPORT)
-
+
```
Comment (gpedro) [inline data/movements/scripts/custom/trainertp.lua]: trailing spaces

**Review by gpedro — CHANGES_REQUESTED** [review]

File: data/globalevents/globalevents.xml (hunk 2 above)
Comment (dudantas) [reply data/globalevents/globalevents.xml]: Yes

**Review by gpedro — APPROVED** [review]
LGTM

**Comment by Sacres97** [comment]
I don't know why it load map with trainers but doesn't load spawn file.

**Pull request merged**